### ISSUE TITLE
[🐛] its NOT deploying on heroku and also not being installed separately as package

### ISSUE THREAD

**Issue by dkhawaja** [juice-shop/juice-shop#1739]
![image](https://user-images.githubusercontent.com/85534387/145571308-6752eb63-af3f-4919-b611-e01940895d85.png)

**Comment by Danbown** [comment]
Hi - to add to the above, when I click 'Deploy app' in Heroku I get the following message.
This is a first-time install:

-----> Building on the Heroku-20 stack
-----> Determining which buildpack to use for this app
-----> Node.js app detected
       
-----> Creating runtime environment
       
       NPM_CONFIG_LOGLEVEL=error
       NODE_VERBOSE=false
       NODE_ENV=production
       NODE_MODULES_CACHE=true
       
-----> Installing binaries
       engines.node (package.json):  12 - 16
       engines.npm (package.json):   unspecified (use default)
       
       Resolving node version 12 - 16...
       Downloading and installing node 16.13.1...
       Using default npm version: 8.1.2
       
-----> Installing dependencies
       Installing node modules (package.json)
       
       > juice-shop@13.0.2 postinstall
       > cd frontend && npm install --legacy-peer-deps && cd .. && npm run build:frontend && (npm run --silent build:server || cd .)
       
       
       > frontend@13.0.2 postinstall
       > ngcc --tsconfig "./src/tsconfig.app.json"
       
       file:///tmp/build_b04f4a22/frontend/node_modules/@angular/compiler-cli/bundles/ngcc/main-ngcc.js:14908
       import { collectExternalReferences, syntaxError as syntaxError2, TypeScriptEmitter } from "@angular/compiler";
                                                                        ^^^^^^^^^^^^^^^^^
       SyntaxError: The requested module '@angular/compiler' does not provide an export named 'TypeScriptEmitter'
           at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21)
           at async ModuleJob.run (node:internal/modules/esm/module_job:181:5)
           at async Promise.all (index 0)
           at async ESMLoader.import (node:internal/modules/esm/loader:281:24)
           at async loadESM (node:internal/process/esm_loader:88:5)
           at async handleMainPromise (node:internal/modules/run_main:65:12)
       npm ERR! code 1
       npm ERR! path /tmp/build_b04f4a22/frontend
       npm ERR! command failed
       npm ERR! command sh -c ngcc --tsconfig "./src/tsconfig.app.json"
       
       npm ERR! A complete log of this run can be found in:
       npm ERR!     /tmp/npmcache.TO2Mr/_logs/2021-12-10T14_35_31_353Z-debug.log
       npm ERR! code 1
       npm ERR! path /tmp/build_b04f4a22
       npm ERR! command failed
       npm ERR! command sh -c cd frontend && npm install --legacy-peer-deps && cd .. && npm run build:frontend && (npm run --silent build:server || cd .)
       
       npm ERR! A complete log of this run can be found in:
       npm ERR!     /tmp/npmcache.TO2Mr/_logs/2021-12-10T14_35_31_430Z-debug.log
-----> Build failed
       
       We're sorry this build is failing! You can troubleshoot common issues here:
       https://devcenter.heroku.com/articles/troubleshooting-node-deploys
       
       If you're stuck, please submit a ticket so we can help:
       https://help.heroku.com/
       
       Love,
       Heroku
       
 !     Push rejected, failed to compile Node.js app.
 !     Push failed

**Comment by bkimminich** [comment]
Fix is on the way, essentially it's an `undo` of v13.0.2 where I pinned `angular-cli` because they had some nasty bug sneak in. This seems now resolved but now the pinned version no longer works with the rest of the Angular 13 ecosystem.

TL;DR: Try again on Heroku, or `git pull` from `master` or use a `v13.0.3` ZIP/Docker image.

**Comment by Danbown** [comment]
Thanks a lot! Working great now :)
